voc/compute.go

```diff
@@ -54,6 +54,10 @@ func (v *VirtualMachineResource) GetLog() *Log {
 	return v.Log
 }
 
+type ContainerResource struct {
+	ComputeResource
```

Comment:
Should the required relationships already be entered here as comment, so that we only have to comment them out and fill in later?